### DISCUSSION TITLE
HTTPS in Flowchart

### DISCUSSION THREAD

**Discussion by vishalduke** [Netflix/conductor#3601] [Q&A]
Hi Everyone,

Need help to call URL's which are HTTPS , currently getting below ERROR :

Failed to invoke HTTP task due to: java.lang.Exception: I/O error on POST request for "[https://localhost:8080/uri":](https://localhost:8080/uri%22:) PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target; nested exception is javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target  
where to configure cert in conductor server

Regards
Vishal Duke

**Comment by rajindersaini** [comment]
How are you running the server. You need to configure it in your jdk. 
for Mac OS  it could be something like this - /Library/Java/JavaVirtualMachines/amazon-corretto-17.jdk/Contents/Home

**Comment by rolik56** [comment]
Get an answer? Same question

**Comment by Robban1980** [comment]
@vishalduke  @rolik56 use the `keytool` to add your certificate to the JVM, conductor is running on. After the certificate has been added restart conductor server and it is supported. 

Here is an example from IBM https://www.ibm.com/docs/en/ztpf/2022?topic=czs-adding-trusted-server-certificates-jvm-truststore-windows-workstation
